variables.tf

```diff
@@ -8,14 +8,22 @@ variable "cluster_name" {
   description = "Name of EKS cluster"
 }
 
+variable "cluster_endpoint" {
+  description = "Endpoint for EKS cluster"
+}
+
+variable "cluster_certificate_authority_data" {
+  description = "CA certificate data for EKS cluster"
+}
+
 variable "helm_namespace" {
   description = "Namespace for Helm chart"
-  default     = "kube-system"
+  default     = "external-secrets"
 }
```

Comment:
What creates this namespace?

Reply:
Added a flag that creates the namespace if the namespace doesn't exist already.